src/poker.js

```diff
@@ -1,10 +1,72 @@
-function winningPair() {}
+/* eslint-disable prettier/prettier */
+function winningPair(pair1, pair2) {
+  if (pair1[0] === pair1[1] && pair2[0] !== pair2[1]) {
+    return pair1
+  } else if (pair2[0] === pair2[1] && pair1[0] !== pair1[1]) {
+    return pair2
+  } else if (pair2[0] === pair2[1] && pair1[0] === pair1[1]) {
```

Comment:
It can be tough to read evaluations like this. I'd recommend storing the result of evaluations into easy to read variables you can use, e.g.

```js
const hand1IsPair = pair1[0] === pair1[1]
const hand2IsPair = pair2[0] === pair2[1]

if (hand1IsPair && hand2IsPair) {
} else if (hand2IsPair && !hand1IsPair) { ...
```